docs/outlook/one-outlook.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Develop Outlook add-ins for the new Outlook on Windows (preview)
+description: Learn about the new Outlook on Windows that's in preview and how to develop Outlook add-ins that are compatible with it.
+ms.topic: article
+ms.date: 02/07/2023
+ms.localizationpriority: medium
+---
+
+# Develop Outlook add-ins for the new Outlook on Windows (preview)
+
+The new Outlook on Windows desktop client unifies the Windows and web codebases to create a more consistent Outlook experience for users and administrators. Its modern and simplified interface has added capabilities and aims to improve productivity, organization, and collaboration for users. More importantly, the new Outlook on Windows supports Outlook web add-ins, so that you can continue to extend Outlook's functionality.
+
+## Impact on VSTO and COM add-ins
+
+Because the new Outlook on Windows is based on the Outlook on the web codebase, VSTO and COM add-ins aren't supported. To ensure your add-in continues to work in the new Outlook on Windows, you must migrate your VSTO or COM add-in to an Outlook web add-in. Migrating to an Outlook web add-in not only enables compatibility with the new Outlook on Windows; it also makes your solution available to users on other platforms, such as Outlook on Mac, on mobile, or on the web.
```

Comment:
```suggestion
Because the new Outlook on Windows is based on the Outlook on the web codebase, VSTO and COM add-ins aren't supported. To ensure your add-in continues to work in the new Outlook on Windows, you must migrate your VSTO or COM add-in to an Outlook web add-in. Migrating to an Outlook web add-in not only enables compatibility with the new Outlook on Windows, it also makes your solution available to users on other platforms, such as Outlook on Mac, on mobile, or on the web.
```